db/schema.rb

```diff
@@ -125,6 +125,13 @@
     t.datetime "updated_at",                          null: false
   end
 
+  create_table "products", force: :cascade do |t|
```

Comment:
it really all depends, I mean the easy solution is to just run db:migrate each time you pull or whatever but gitignore might be the good choice for now since none of the data is permanent